### ISSUE TITLE
Calculate total resources for kubernetes pods

### ISSUE THREAD

**Issue by ianstanton** [astronomer/astronomer#284]
* Calculate total request & limit resources for kubernetes pods
* This will help determine minimum cluster requirements

**Comment by ianstanton** [comment]
| Pod                        | Request CPU  | Request Mem  | Limit CPU  | Limit Mem  |
|-------------------------|--------------|---|---|---|
| `orbit`                 | 100m         | 256Mi  | 500m  | 1024Mi  |
| `houston`               | 250m         | 512Mi  | 800m  | 1024Mi  |
| `prisma`                | 250m         | 512Mi  | 500m  | 1024Mi  |
| `commander`             | 250m         | 512Mi  | 500m  | 1024Mi  |
| `registry`              | 250m         | 512Mi  | 500m  | 1024Mi  |
| `install`               | 100m         | 256Mi  | 500m  | 1024Mi  |
| `nginx`                 | 500m         | 1024Mi  | 1  | 2048Mi  |
| `grafana`               | 250m         | 512Mi  | 500m  | 1024Mi  |
| `prometheus`            | 1000m        | 4Gi  | 1000m  | 4Gi  |
| `elasticsearch client`  | 1            | 2Gi  | 2  | 4Gi  |
| `elasticsearch data`    | 1            | 2Gi  | 2  | 4Gi  |
| `elasticsearch master`  | 1            | 2Gi  | 2  | 4Gi  |
| `kibana`                | 250m         | 512Mi  | 500m  | 1024Mi  |
| `fluentd`               | 250m         | 512Mi  | 500m  | 1024Mi  |
| `kubeState`             | 250m         | 512Mi  | 500m  | 1024Mi  |
| Total                   |           |   |   |   |

**Comment by ianstanton** [comment]
@schnie I noticed `heapMemory: "2g"` for each `elasticsearch` pod. Is this an extra `2Gi` in addition to the requested memory?

**Comment by schnie** [comment]
@ianstanton nope that is just a separate setting that I think might get passed as a jvm argument or something. We just want them to match what we actually give it. JVM wants 2g, and kube wants 2Gi, so they're currently separate settings due to the format inconsistency there.

**Comment by vparekh94** [comment]
@ianstanton this is great - do you think we should put this in docs somewhere? Maybe in a section labelled "Modifying Resources" that would show a modified `config.yaml` of changing a service's default resource allocation?

If so, can you open an issue and assign it to me? Would also be cool if you wrote it as a user story and see if we can use that as a template for ongoing docs issues.

**Comment by ianstanton** [comment]
Yeah I like that idea. Schnie mentioned adding it to the `astronomer/helm.astronomer.io` README. I think it  would be worth having it in `/docs` as well!
